docs/releases/1.11.txt

```diff
@@ -101,6 +101,11 @@ Minor features
 * :func:`~django.contrib.auth.update_session_auth_hash` now rotates the session
   key to allow a password change to invalidate stolen session cookies.
 
+* The new attribute ``success_url_allowed_hosts`` for
+  :class:`~django.contrib.auth.views.LoginView` and
+  :class:`~django.contrib.auth.views.LogoutView` allows specifying a set of
+  hosts that are safe for redirect after login and log out.
```

Comment:
safe for redirecting to
is login/logout one word or two? (or are they really different?)